src/NewField.cpp

```diff
@@ -277,20 +283,28 @@ void NewFieldColumn::take_over_mods(NewFieldColumn& other)
 #undef CPY
 }
 
-void NewFieldColumn::set_defective_mode()
+void NewFieldColumn::set_defective_mode(bool mode)
```

Comment:
I know we use `bool` types for flags often, but a part of me feels this would better fit as an enumeration.

This is not something that has to change yet. Just another note of something I want to eventually take care of.